.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test Homebrew install
+    runs-on: macos-latest
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install macFUSE
+        run: brew install --cask osxfuse
+
+      # - name: Debug
+      #   uses: mxschmitt/action-tmate@v3
```

Comment:
are you still working on that or is it a remainder that should be removed for now?

Reply:
I think it should stay there if someone wants to debug the workflow in the future.

Reply:
ok